frontend/src/worksheets/TableListOfWorkSheets.js

```diff
@@ -100,6 +102,13 @@ export default function TableListOfWorkSheets() {
                           onClick={() => closeAndReload(worksheet)}
                         />
                       )}
+                      {worksheet.worksheetStatus !== 'CREATED' && (
+                        <Link onClick={() => WorkSheetPDF(worksheet)}>
```

Comment:
Ez semmiképpen ne legyen Link, az csak a routing-os linkelésnél jó. A PdfButton elég lenne és annak adni onClick-et. Ahhoz, hogy ez működjön ahhoz viszont a PdfButton-ön belül le kell kezelni ezt a prop-ot, praktikus tovább kell adni a Printernek.

Reply:
Ez is folyamatban van. :)

Reply:
Módosítva.